learn/app-development/widgets/prefab/enterprise-marketplace.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Enterprise Marketplace"
+id: "enterprise-marketplace"
+---
+---
+
+## Enterprise Marketplace
+
+Enterprise Marketplace lets teams published artifacts for reuse across the team. Developer create artifacts and team admin approves and publishes the version of artifact to Marketplace via Teamportal. 
+
+![Enterprise Marketplace](/learn/assets/wm_enterprisemarketplace.png)
+
+You can view the list of prefabs to be published to Enterprise Marketplace in Manage Prefabs page.
+
+![Enterprise Marketplace](/learn/assets/wm_marketplaceprefabs.png)
+
+:::note
```

Comment:
We should define the roles involvled in this activity clearly before using them in sentence.